easybuild/easyconfigs/l/likwid/likwid-4.2.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'likwid'
+version = '4.2.0'
+
+homepage = 'http://code.google.com/p/likwid/'
+
+description = """
+ Likwid stands for Like I knew what I am doing. This project contributes easy
+ to use command line tools for Linux to support programmers in developing high
+ performance multi threaded programs.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# https://github.com/RRZE-HPC/likwid/archive/4.2.0.tar.gz
+source_urls = ['https://github.com/RRZE-HPC/likwid/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['1cb68f895cfce83b2d08a67bc4a0f8628202b59f2ea5949ef12061f3e7b03e94']
+
+patches = ['likwid-%(version)s-config-mk.patch']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+skipsteps = ['configure']
+
+buildopts = 'CC="$CC" CFLAGS="$CFLAGS -std=c99" PREFIX=%(installdir)s'
+
+installopts = 'PREFIX=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ["bin/likwid-memsweeper", "bin/likwid-mpirun", "bin/likwid-perfctr",
+              "bin/likwid-perfscope", "bin/likwid-pin", "bin/likwid-powermeter",
+              "bin/likwid-topology", "lib/liblikwidpin.%s" % SHLIB_EXT,
+              "lib/liblikwid.%s" % SHLIB_EXT],
+    'dirs': ["man/man1"]
+}
+
+maxparallel = 1
```

Comment:
@JackPerdue please move this up, right below `buildopts` (we try to group easyconfig parameters per installation step)